docs/index.rst

```diff
@@ -207,45 +209,10 @@ This command will generate a ``PostFactory`` class that looks like this:
                 zenstruck_foundry:
                     make_factory:
                         default_namespace: 'App\\MyFactories'
-                    make_story:
-                        default_namespace: 'App\\MyStories'
 
     You can override this configuration by using the ``--namespace`` option.
 
-
-.. note::
```

Comment:
I think this is no longer applicable - @nikophil, can you confirm?

Reply:
Actually this is still true. If you have a `@method` which basically is a lie, you'll need the `@phpstan-method` annotation.

But I've just tested with foundry 2 + phpstorm 2023.3, and autocomplete comes out of the box, even with `PersistentProxyObjectFactory`

I think this is because phpstorm really improved how it handles generics + we now return `@return T&Proxy<T>` which is easier to understand than the old `Proxy<T>` which is a `@mixin` of a template, which phpstorm notoriously does not handle very well.

So... maybe we should remove all the `@method` even in doc? On the other hand, not anybody uses phpstorm. So we can document both `@method` and `@phpstan-method` in a `.. note::`: "if autocomplete does not work out of the box, you can still add `@method`, but be sure to add `@phpstan-method` as well..."?

I think we should also remove these phpdoc from the `make:factory` command, and make it optional with an option `--add-phpdocs`. WDYT?

Reply:
I've made the change here: https://github.com/zenstruck/foundry/pull/611